tutorials/Applciation.getCurrent.md

```diff
@@ -0,0 +1,18 @@
+Returns an instance of the currently running application.
+# Example
+```js
+async function getCurrent() {
+    const app = await fin.Application.create({
+        uuid: 'app-1',
+        name: 'myApp',
+        url: 'https://openfin.co',
+        autoShow: true
+    });
+
+    await app.run();
+    await app.getCurrent();
```

Comment:
This demo is using the application class getCurrent. Lets write the tutorial for the application module getCurrent.

Similar to this:
http://cdn.openfin.co/jsdocs/stable/tutorial-application.getCurrent.html

And make a note that it will only work in OpenFin, not in the nodeJs environment.